sql/updates/world/master/2021_02_02_00_world_priest_prayer_of_mending.sql

```diff
@@ -0,0 +1,5 @@
+DELETE FROM `spell_script_names` WHERE `spell_id` IN (33110, 33076, 41635, 155793);
```

Comment:
is 33110 an old spellid related to prayer of mending?

Reply:
Exists both on wowhead (not in Classic) & TC ubuntu aowow (3.3.5), not sure of the history
- https://www.wowhead.com/spell=33110/prayer-of-mending
- https://tcubuntu.northeurope.cloudapp.azure.com/aowow/?spell=33110
- https://wow.gamepedia.com/Prayer_of_Mending

Reply:
BTW, there used to be a rule that
```sql
DELETE FROM `spell_script_names` WHERE `ScriptName` IN (
```
is the preferred method. Many have debated the validity of that rule.

Reply:
Unless there is a good & obvious reason for deleting by spell ID, TC prefers using
```sql
DELETE FROM `spell_script_names` WHERE `ScriptName` IN ('spell_pri_prayer_of_mending', 'spell_pri_prayer_of_mending_aura', 'spell_pri_prayer_of_mending_jump');
```